datafusion/expr/src/logical_plan/display.rs

```diff
@@ -221,6 +229,488 @@ impl<'a, 'b> TreeNodeVisitor for GraphvizVisitor<'a, 'b> {
     }
 }
 
+/// Formats plans to display as postgresql plan json format.
```

Comment:
I suggest we add a link here to the spec of the postgresql plan format

Reply:
I've mentioned that there is no formal spec for this format. But  it seems that the format is relative simple, it mainly requires `Node Type` as name, `Output` as schema, and `Plans` property as children.  Others are not required and visualizer can display them in misc.